docs/incompatible_changes/3.5.rst

```diff
@@ -0,0 +1,756 @@
+            if case.message is None:
+                await ctx.send("No message is available for this case.")
+                return
+
+            await ctx.send(
+                "People reacted to this modlog case with: "
+                + ", ".join(case.message.reactions)
+            )
+
+After:
+
+.. code:: python
+
+    import discord
+    from redbot.core import commands
+
+    class MyCog(commands.Cog):
+        @commands.guild_only()
+        @commands.command()
+        async def command(self, ctx, case_number: int):
+            case = await modlog.get_case(case_number, ctx.guild, ctx.bot)
+            if case.message is None:
+                await ctx.send("No message is available for this case.")
+                return
+
+            try:
+                msg = await case.message.fetch()
+            except discord.NotFound:
+                await ctx.send("No message is available for this case.")
+            else:
+                await ctx.send(
+                    "People reacted to this modlog case with: "
+                    + ", ".join(msg.reactions)
+                )
+
+``redbot.core.bot.RedBase`` has been merged into ``redbot.core.bot.Red``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Historically, ``RedBase`` existed to allow using Red for self/user bots back when
+it was not against Discord's Terms of Service. Since this is no longer a concern,
+everything from ``RedBase`` have been moved directly to `Red` and ``RedBase`` class
+has been removed.
+
+If you were using ``RedBase`` for runtime type checking or type annotations,
+you should now use `Red` instead. Since both of these classes resided in the same
+module, it should be a matter of simple find&replace.
+
+``Context.maybe_send_embed()`` requires content with length of 1-2000 characters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Context.maybe_send_embed()` now requires the message's length to be
+between 1 and 2000 characters.
+
+Since the length limits for regular message content and embed's description are
+different, it is easy to miss an issue with inappropriate handling of length limits
+during development. This change should aid with early detection of such issue by
+consistently rejecting message with length that can't be used with
+both embed and non-embed message.
+
+This change only affects code that is already not guaranteed to work.
+You should make sure that your code properly handles message length limits.
+
+``menu()`` listens to both reaction add and remove
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Listening only to reaction add results in bad user experience.
+If the bot had Manage Messages permission, it removed the user's reaction
+so that they don't have to click twice but this comes with a noticable delay.
+This issue is even more noticable under load, when the bot ended up hitting
```

Comment:
```suggestion
so that they don't have to click twice but this comes with a noticeable delay.
This issue is even more noticeable under load, when the bot ended up hitting
```